src/test/test-utils.ts

```diff
@@ -118,7 +118,15 @@ export async function runScannerOnContents(
     scanner: Scanner<ParsedDocument, any, any>, url: string, contents: string) {
   const overlayLoader = new InMemoryOverlayUrlLoader();
   const analyzer = new Analyzer({urlLoader: overlayLoader});
-  overlayLoader.urlContentsMap.set(analyzer.resolveUrl(url), contents);
+  overlayLoader.urlContentsMap.set(analyzer.resolveUrl(url)!, contents);
   const {features, warnings} = await runScanner(analyzer, scanner, url);
-  return {features, warnings, analyzer};
+  return {features, warnings, analyzer, urlLoader: overlayLoader};
+}
+
+export function fUrl([text]: TemplateStringsArray): FileRelativeUrl {
```

Comment:
```
export function fileRelativeUrl
```

explicit function names > saving characters. I believe this is a team value we've already discussed. As someone who didn't have all our Url types in my head I had no idea what this was doing when I saw it used.

Also, jsdoc